Add known issue to NT (#8030)
mendix/docs: content/en/docs/releasenotes/mobile/native-template/nt-studio-pro-9-parent/nt-7-rn.md

description: "Native Template 7 release notes."
---

## 7.0.14 {#714}

**Release date: June 28, 2024**

### Fixes

> We have fixed an issue with cookies encryption. In case the app still does not load correctly after updating to this version into a problem, please clear the app cache and reload the app.

Reviewer: ```suggestion
We have fixed an issue with cookies encryption. If the app still does not load correctly after updating to this version into a problem, please clear the app cache and reload the app.
```

## 7.0.13 {#713}

**Release date: June 5, 2024**

### Improvements

* We enhanced encrypted file sync on Android.

> Known issue: In this version we addressed a cookie encryption security issue which turned out to be in compatible with the existing mechanism.

Reviewer: ```suggestion
Known issue: In this version we addressed a cookie encryption security issue which turned out to be incompatible with the existing mechanism.
```
